invenio_records_lom/utils/__init__.py

```diff
@@ -0,0 +1,439 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 Graz University of Technology.
+#
+# invenio-records-lom is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Utilities for creation of LOM-compliant metadata."""
+
+# allow `list[str]` annotation, PEP585 makes this unnecessary for python>=3.9
+from __future__ import annotations
+
+import copy
+import typing as t
+from collections.abc import MutableMapping
+from pathlib import Path
+
+
+class DotAccessWrapper(MutableMapping):
```

Comment:
please don't add class definitions to __init__.py file. __init__.py file should only contain the __all__ statement (with the for it necessary rest)